botcore/verify.py

```diff
@@ -95,170 +109,201 @@ def fetch_verifying(self):
     async def on_message(self, message):
         member = self.guild.get_member(message.author.id)
         if member.id in self.verifying:
-            await self.state_handler[self.verifying[member.id].state](member, message)
+            state = self.verifying[member.id]["_state"]
+            await self.state_handler[state](member, message)
 
     def get_code(self, user):
         msg = bytes(str(user.id), "utf8")
         return hmac.new(self.secret, msg, "sha256").hexdigest()
 
-    async def verify_proc_begin(self, member):
+    async def proc_begin(self, member):
         if member.id in self.verifying:
-            await member.send("You are already undergoing the verification process. To restart, type `!restart`.")
+            await member.send("You are already undergoing the verification "
+                "process. To restart, type `!restart`.")
```

Comment:
Hardcoding the command prefix even though it exists in your config? I would just remove the config option, we're not changing the prefix anytime soon.